README.md

```diff
@@ -19,3 +20,7 @@ to be the most efficient one!
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## Runtime Analysis of my implementation
+
+The complexity of my implementation would be $\Theta(n^3)$ where $n$ is the number of nodes in the given graph. I found the complexity from the fact that the first for loop in the graph will loop $n$ times as it must add a distance for each node, the while loop will also run $n$ times as it runs until every node has been marked. So far this gives us $n + n$ complexity. The while loop has nested for loops that will run $n$ to $(n-k)$ times as each node can only be connected to at most $n$ nodes (all nodes plus itself) or $(n-k)$, $k$ being the number of edges the given node has. This gives us a complexity of $n + (n*(n-k)*(n-k))$ or $n + n^3$. Thus the overall complexity of this implementation is $\Theta(n^3)$
```

Comment:
But not all of these loops are nested. Don't see how you get to $|V|^3$ with your code.